roles/keycloak/tasks/deploy.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: "Create configMap with TLS certificates"
+  kubernetes.core.k8s:
+    state: present
+    kubeconfig: /share/kubeconfig
+    definition: "{{ lookup('file', keycloak_configuration_directory + '/configmap-certs.yml') | from_yaml_all }}"
```

Comment:
I am actually wondering, what is the configmap-certs.yml? It is not part of the PR, not in the repo, not present in the documentation

Reply:
This is a remnant of the first iteration, this was discarded but I didnt clean up all it seems